frontend/views/containers/chatroom/file-attachment/ChatFileAttachmentWarningModal.vue

```diff
@@ -4,10 +4,10 @@ modal-template(
   :a11yTitle='L("Chat attachment too large modal")'
 )
   template(slot='title')
-    span {{ warningMessages.title }}
+    i18n File too large
 
   .c-content
-    span {{ warningMessages.content }}
+    i18n(:args='{ sizeLimit: config.sizeLimit }') That file is too large and cannot be uploaded. The limit is {sizeLimit} MB
```

Comment:
Think this will show `6291456 MB`, which I think is wrong 😆

Reply:
@taoeffect Right, good catch. Fixed it.

<img src='https://github.com/okTurtles/group-income/assets/17641213/e9bdb2bc-e848-4112-9a46-d13c237ccf4b' width='440'>